WordPress/gutenberg
# Fix navigate regions backwards for macOS Firefox and Safari.

Noticed while testing https://github.com/WordPress/gutenberg/pull/44883

## What?

When pressing the keys Shift + Control + backtick to navigate backwards through the editor regions, the returned `event.key` varies across macOS browsers:
- Chrome: event.key is the backtick 
- Firefox: event.key is the tilde `~`
- Safari: event.key is the tilde `~`

My environment:
- macOS Monterey 12.6
- Firefox 105.0.3
- Safari Version 16.0
- Keyboard layout set to English

This makes region navigation fail on macOS Firefox and Safari, as the event.key is expected to match the backtick defined in the editor keyboard shortcuts.
 
## Why?

Navigation through the editor regions is broken and needs to be fixed.

## How?

Instead of trying to fix browser inconsistencies, seems to me the quickest way to fix this is by defining one more shortcut alias for the `Navigate to the previous part of the editor` command.

## Testing Instructions





**Test with Firefox or Safari on macOS. Make sure to test with an keyboard with an English layout or set your keyboard layout to English, as the backtick key may not be present on other keyboard layouts.**

Before:
- Go to the Post editor or to the Site editor.
- Click on any empty part of the interface to make sure the editor has focus.
- Press the key combination Shift + Control + backtick
- Observe that nothing happens.

After:
- Repeat the steps above.
- Observe the keyboard shortcut works as expected, moving focus to the previous editor region.
- Test also with Chrome, to make sure everything works as expected.


## Screenshots or screencast
